toolchains/xeon.persee/debug_toolchain.cmake

```diff
@@ -0,0 +1,14 @@
+
+include(${CMAKE_CURRENT_LIST_DIR}/../cicd_default_toolchain.cmake)
+
+# CMake options
+set(CMAKE_BUILD_TYPE Debug)
+
+# Compiler options
+set(CMAKE_CXX_FLAGS "-Wall -Wno-sign-compare -Wno-unused-but-set-variable")
+
+# Activate/deactivate parts of the code
+if (DEFINED ENV{DDC_BUILD_TESTING})
+    set(DDC_BUILD_TESTS $ENV{DDC_BUILD_TESTING})
+endif()
+set(ACTIVATE_RESTART_TESTS OFF)
```

Comment:
I copy-pasted this part, I don't know if it should be added or not

Reply:
Where did you copy/paste it from?

Reply:
From `v100.persee/toolchain.cmake` that actually should be called `release_toochain.cmake`.

Reply:
Ok. I think the aim of this block was to use the environment variable set here:
https://github.com/gyselax/gyselalibxx/blob/main/.github/actions/test_filter/action.yml#L38-L39

to run GPU tests for DDC. I don't think it is relevant here

Reply:
I will remove it then

Reply:
Ok. I have removed the "Ready to review" label for now. Please put it back when you have updated this

Reply:
done in 075d70a41aaf5550daa7cb021110d4a0d1b1e170 with renaming of `release_toolchain` to follow the convention of other toolchains.